pages/[country].js

```diff
@@ -36,6 +36,9 @@ export const getStaticProps = async ({ params: { country } }) => {
   const resCountries = await fetch("https://covid19.mathdro.id/api/countries");
   const { countries } = await resCountries.json();
 
+  const resPerDay = await fetch("https://covid19.mathdro.id/api/daily");
```

Comment:
We need to show the stats per day by country not only for the whole world

Reply:
Thank you for explaining this @smakosh 
I'll try to fix it is asap